swagger/v1/component_schemas/ParticipantDeclaration.yml

```diff
@@ -18,5 +18,5 @@ example:
     attributes:
       participant_id: db3a7848-7308-4879-942a-c4a70ced400a
       declaration_type: started
-      declaration_date: 2021-05-31T02:21:32Z
+      declaration_date: "2021-05-31T02:21:32.000Z"
```

Comment:
Why? We're trying not to show these. I know swagger can generate them, but we shouldn't be going down to the millisecond level.